README.md

```diff
@@ -1,3 +1,14 @@
 ## Public GitHub, GitLab, and Bitbucket projects Scala Steward is keeping up-to-date
 
 See <https://github.com/scala-steward-org/scala-steward> for more info.
+
+## Terms of service
+
+This public Scala Steward instance is operated and paid for by [@fthomas](https://github.com/fthomas) for the benefit of the Scala community.
+I reserve the right to not support projects I deem unethical or otherwise inappropriate.
+
+If your project is about one of these topics, it will be rejected or removed:
+
+* Bitcoin or other cryptocurrencies
```

Comment:
I'd like to suggest pointing out PoW blockchains specifically

Reply:
Sounds good to me. Do you have a suggestion how the above bullet point should be changed?

Reply:
"Bitcoin or any other Proof of Work cryptocurrencies"? I'm any case I'm happy to merge

Reply:
I think this is a bit too specific for my taste. I also don't want to support proof of space cryptocurrencies that waste hard drives. If there ever will be cryptocurrencies that are not scams, I'm happy to refine this point. :-)